src/compiler/compiler.rs

```diff
@@ -471,7 +466,7 @@ where
     };
 
     debug!("[{}]: Attempting distributed compilation", out_pretty);
-    let compile_out_pretty = out_pretty.clone();
+    let _compile_out_pretty = out_pretty.clone(); // TODO: double check if we want to call two times in two lines in a row like this
```

Comment:
Just delete this line. It has no purpose if it still compiles after adding the `_` prefix.